generators/server/templates/src/main/resources/config/application.yml.ejs

```diff
@@ -119,8 +119,10 @@ management:
     health:
       show-details: when_authorized
       roles: "ROLE_ADMIN"
-    jhimetrics:
-      enabled: true
+      probes:
+        enabled: true
+      jhimetrics:
+        enabled: true
```

Comment:
`jhiMetrics` is now under `endpoint.health`, but before it was directly under `endpoint` - is this intended change or should `jhiMetrics` still be under `endpoint`?
```suggestion
    jhimetrics:
      enabled: true
```

Reply:
I believe you are correct that it's indented too far. I'll commit your suggestion.